xExtension-ImageProxy/README.md

```diff
@@ -67,3 +67,46 @@ RewriteRule ^ http://%1 [QSA,P,L]
   Require all denied
 </LocationMatch>
 ```
+
+### nginx configuration
+
+In order to use nginx's [proxy
+module](https://nginx.org/en/docs/http/ngx_http_proxy_module.html), you will
+need to set the following settings:
+
+* `proxy_url` = **https://www.example.org/proxy?key=changeme&url=**
+* `scheme_include` = **1**
+* `url_encode` = **0**
+
+Add this to your nginx config:
+
+``` nginx
+# Use 1 GiB cache with a 1 MiB memory zone (enough for ~8,000 keys).
+# Delete data that has not been accessed for 30 minutes.
```

Comment:
30 minutes is for a very busy server? It doesn't matter much since it's just an example but I'd be more inclined to put in a value closer to, let's say, 36h, so I'm curious about the rationale behind 30 minutes in any case. :-)

Reply:
I didn't put too much thought in it to be honest. Theoretically images could change after publishing, but I guess that rarely happens. How about I change it to delete after 12 hours if inactive, 2 days maximum? I think most feed articles are read within a day, so I don't see the point in keeping it much longer.

Reply:
Sure. ^_^